process/database/elasticSearchConnector.go

```diff
@@ -151,6 +150,32 @@ func (esc *elasticSearchConnector) doSearchRequest(query object, index string, s
 	return decodedBody, nil
 }
 
+func (esc *elasticSearchConnector) doSearchRequestTx(address string, index string, size int) (object, error) {
```

Comment:
Yup, makes more sense to add query parameters in the URL. The fact that `esc.client.Search` performs a GET with `body` under the hood is somehow surprising behavior - and lead to this issue.

I am linking this extra SO discussion, related to this matter:

 - https://stackoverflow.com/questions/36939748/elasticsearch-get-request-with-request-body